node/send_receive/package.json

```diff
@@ -29,7 +32,7 @@
   },
   "homepage": "https://github.com/Azure/azure-event-hubs#readme",
   "scripts": {
-    "lint": "jshint --show-non-errors .",
+    "lint": "jshint --show-non-errors . && tslint -c ./tslint.json \\\"./lib/**/*.ts\\\"",
```

Comment:
Typescript is installed localy in node_modules and this script will work without the user needing to install typescript globally.